build.gradle

```diff
@@ -17,8 +17,8 @@ dependencies {
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
 	// Fabric API. This is technically optional, but you probably want it anyway.
-	modImplementation "net.fabricmc.fabric-api:fabric-api-base:${project.api_base_version}"
-	include "net.fabricmc.fabric-api:fabric-api-base:${project.api_base_version}"
+	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.api_base_version}"
+	include "net.fabricmc.fabric-api:fabric-api:${project.api_base_version}"
```

Comment:
This is not correct. i was including specific subsections of fabric api, and this is the whole thing

Reply:
This was quick and dirty fix to get to build, I believe there was some problem with just using fabric-api-base. lmgbty